### PR TITLE
Fix eslint rules inside apps/common-app

### PR DESCRIPTION
**Pull request by Latropos**
## Summary
When introducing monorepo we unified all the eslint rules across the project.
That's why we the eslint rules got more restrictive and we had to fix all the existing problems.

## Test plan
Run `yarn lint` inside apps/common-app

### CODE REVIEW

File: apps/common-app/src/examples/ShareablesExample.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ function RemoteNamedFunctionSyncCallDemo() {
 
 function RemoteAnonymousFunctionSyncCallDemo() {
   const handlePress = () => {
+    // eslint-disable-next-line no-constant-condition
     const foo = true ? () => {} : () => {};
```
Comment (Latropos) [inline apps/common-app/src/examples/ShareablesExample.tsx:66]: @tomekzaw I have no idea why do we need this constant condition...

File: apps/common-app/src/examples/SharedElementTransitions/ImageStack.tsx [hunk 2]
```diff
@@ -57,7 +55,7 @@ export function ImageStack({
                 },
                 styles.roundedBorder,
               ]}
-              sharedTransitionTag={`${image}@${index}`}
+              sharedTransitionTag={`SET${index}`}
```
Comment (Latropos) [inline apps/common-app/src/examples/SharedElementTransitions/ImageStack.tsx:58]: `${image}` evaluates to string "[Object Object]", I've tested that the gallery example still works

**Review by piaskowyk — APPROVED** [review]
For me is ok, but let's wait for @tomekzaw review

File: apps/common-app/src/examples/WorkletRuntimeExample.tsx [hunk 3]
```diff
@@ -74,7 +74,7 @@ function CreateWorkletRuntimeDemo() {
     const runtime = createWorkletRuntime('foo');
     console.log(runtime);
     console.log(runtime.name);
-    console.log(`${runtime}`);
+    console.log(`${runtime.name}`);
```
Comment (tomekzaw) [inline apps/common-app/src/examples/WorkletRuntimeExample.tsx:77]: This is the same as `console.log` above. The purpose of this `console.log` was to check the output of `${runtime}`. Can we restore the original `console.log`?

File: apps/common-app/src/examples/WorkletFactoryCrashExample.tsx [hunk 4]
```diff
@@ -6,8 +6,9 @@ export default function App() {
   function handleOnPress() {
     function badWorklet() {
       'worklet';
-      // @ts-expect-error
-      unexistingVariable;
+      // @ts-expect-error This is an expected error
+      // eslint-disable-next-line no-unused-expressions
+      unExistingVariable;
```
Comment (tomekzaw) [inline apps/common-app/src/examples/WorkletFactoryCrashExample.tsx:11]:
```suggestion
      nonExistentVariable;
```

File: apps/common-app/src/examples/ShareablesExample.tsx (hunk 1 above)
Comment (tomekzaw) [reply apps/common-app/src/examples/ShareablesExample.tsx:66]: Me neither but let's leave it as it is now

**Review by tomekzaw — APPROVED** [review]
LGTM

**Pull request merged**